### PR TITLE
Update animals table

### PR DESCRIPTION
**Pull request by rmrmrmr**
# On this pull request🗃️:

- Added a `species` column of type `string` to the `animals` table & modified the corresponding `schema.sql` file 🐺🐱.
- Inserted data for seven new animals & modified the corresponding `data.sql` file.
- Added queries on the `queries.sql` for the following transactions ↔️✅
    - Updated the `animals` table by setting the `species` column to `unspecified`.  Verified the change was made and rolled back the transaction.
    

    - Updated the `animals` table by setting the `species` column to `digimon` for all animals with a name ending in `mon`. Then, updated the `animals` table by setting the `species` column to `pokemon` for all animals that don't have `species` already set. Finally, we committed the transaction and verified that the change persisted.
    

    - Took a deep breath🌬️.... and deleted all records in the `animals` table. Then we rolled back the transaction. We then verified all records still existed and kept on breathing 🥳
    

    - Deleted all animals born after Jan 1st, 2022, and created a savepoint. We then updated all animals' weights to be the corresponding negative number and rolled back to the save point. Finally, we updated all negative animal weights to be their related positive part and committed the transaction. 
    

- Added media queries on the `queries.sql` file to answer the following questions:
    - How many animals are there?
    - How many animals have never tried to escape?
    - What is the average weight of animals?
    - Who escapes the most, neutered or not neutered animals?
    - What is the minimum and maximum weight of each type of animal?
    - What is the average number of escape attempts per animal type of those born between 1990 and 2000?

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -4,3 +4,8 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VAL
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Gabumon', '2018-11-15', 2, true, 8.0);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Devimon', '2017-05-12', 5, true, 11.0);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Charmander', '2020-02-08', 0, false, 11.0);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Plantmon', '2021-11-15', 2, true, 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Squirtle', '1993-04-02', 3, false, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Angemon', '2005-06-12', 0, true, 45.0);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Boarmon', '2005-06-07', 7, true, 20.4);
```
Comment (t-yanick) [inline data.sql]: - [x] Kindly check the screenshot below and insert the data as per project requirements. For example I cannot find the animal with name Ditto inserted in your database. Looking also at the screenshot you will notice some weights are negative values.
Screen Shot 2023-02-21 at 7 36 10 PM

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @rmrmrmr , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Screenshots added to PR description
:heavy_check_mark: SQL commands put in the correct files
:heavy_check_mark: Great PR description

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x] Great job working on this milestone. Your MIT license link is broken and leads to an error 404 page when clicked. Kindly fis this as it not professional to allow broken links in our codebase.
Screen Shot 2023-02-21 at 7 27 50 PM

- [x] There is this text `NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.`
It is just a guide that comes with the template on how to add the license link to your ReadMe. Kindly remove it.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (rmrmrmr) [reply data.sql]: Done 🚀

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @rmrmrmr 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**